### ISSUE TITLE
Better feedback/hud indicator/blur effect direction for wall crashes

### ISSUE THREAD

**Issue by darkgriffin1986** [virtexedge/metric-racer-sdk#19]
When driving narrow turns or s bend areas and hitting a wall, it can be tricky to tell from the screen blur which rail side the collision happened.  Perhaps the HUD could flash in the direction of damage with a radial indicator, or the blur could emerge from the side of the screen where the hit was instead of over the entire screen (left or right instead of current behavior).  This would help me know which way to steer or expect the ship to bounce on collision, and also help the blur/shatter feel like it came from that wall/rail collision instead of potentially other sources such as opponent shots.  The game could give better feedback for drivers learning the walls of a new track layout this way.

**Comment by virtexedge** [comment]
Agreed, that would help the player know what they hit and where they hit as well.
